MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -100,53 +100,60 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate,
     
     private func showAnswerResult(isCorrect: Bool) {
         if isCorrect {
-            corrrectAnswers += 1
+            correctAnswers += 1
         }
         
-            imageView.layer.masksToBounds = true
-            imageView.layer.borderWidth = 8
-            imageView.layer.cornerRadius = 20
-            imageView.layer.borderColor = isCorrect ? UIColor.customGreen.cgColor : UIColor.customRed.cgColor
-            self.yesButton.isEnabled = false
-            self.noButton.isEnabled = false
-
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
+        imageView.layer.cornerRadius = 20
+        imageView.layer.borderColor = isCorrect ? UIColor.customGreen.cgColor : UIColor.customRed.cgColor
+        self.yesButton.isEnabled = false
+        self.noButton.isEnabled = false
+        
         
         DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) { [weak self] in
             guard let self = self else {return}
             // запускаем задачу через 1 секунду
             self.yesButton.isEnabled = true
             self.noButton.isEnabled = true
             self.imageView.layer.borderWidth = 0
-            self.showNextQestionOrResults()
-           
-
+            self.showNextQuestionOrResults()
+            
+            
         }
         
     }
     
-    private func showNextQestionOrResults() {
+    private func showNextQuestionOrResults() {
         
         if currentQuestionIndex == questionsAmount - 1 {
             
             if statisticService.gamesCount >= 0 {
                 statisticService.gamesCount += 1
             }
             
-            let record = GameRecord(correct: corrrectAnswers, total: questionsAmount, date: Date().dateTimeString)
-            statisticService.store(correct: corrrectAnswers, total: questionsAmount)
-            statisticService.bestGame = record.compareRecord(current: record, previous: statisticService.bestGame)
+            let record = GameRecord(correct: correctAnswers, total: questionsAmount, date: Date().dateTimeString)
+            
+            statisticService.store(correct: correctAnswers, total: questionsAmount)
+            
+            let bestRecord = GameRecord.returnBestRecords(current: record, previous: statisticService.bestGame)
```

Comment:
булевы переменные принято называть с приставкой `is`, и в сфивт стараются называть функции так, чтобы получалось предложение :) не уверена, что предлагаю идеальный вариант, но можно к этому стремиться
```suggestion
            let isBestRecord = GameRecord.isBest(current: record, previous: statisticService.bestGame)
```